packages/web-client/app/components/card-space/create-space-workflow/index.hbs

```diff
@@ -24,21 +24,22 @@
 
 
 {{#unless this.workflow.isCanceled}}
+  {{!-- CardSpace::ProfileCard --}}
```

Comment:
Added this comment so it's easier to find the `CardSpace::ProfileCard` component when searching the codebase